### PR TITLE
Add a catch for missing sdrf file, test for bulk

### PR DESCRIPTION
**Pull request by lingyun1010**

### CODE REVIEW

File: src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java [hunk 1]
```diff
@@ -23,6 +23,48 @@
...
+            populator.execute(dataSource);
+        }
+
+        @ParameterizedTest
+        @MethodSource("expressionAtlasExperimentsProvider")
+        void testParserForExpressionAtlas(String experimentAccession) {
+            SdrfParser sdrfParser = new SdrfParser(new DataFileHub(dataFilesPath.resolve("gxa")));
+
+            assertThat(sdrfParser.parseSingleCellTechnologyType(experimentAccession)).isEmpty();
+
```
Comment (alfonsomunozpomer) [inline src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java]: unnecessary blank line.

File: src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java [hunk 2]
```diff
@@ -23,6 +23,48 @@
...
+        @Inject
+        private Path dataFilesPath;
+
+        @Inject
+        private JdbcUtils jdbcUtils;
+
+        @BeforeAll
+        void populateDatabaseTables() {
+            ResourceDatabasePopulator bulkPopulator = new ResourceDatabasePopulator();
```
Comment (alfonsomunozpomer) [inline src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java]: For consistency/simplicity you can name this `bulkPopulator`. I think this name comes from a time before nested tests. Also, you can use `var populator = ...`.

File: src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java [hunk 3]
```diff
@@ -23,6 +23,48 @@
...
+            bulkPopulator.addScripts(new ClassPathResource("fixtures/gxa-experiment-fixture.sql"));
+            bulkPopulator.execute(dataSource);
+        }
+
+        @AfterAll
+        void cleanDatabaseTables() {
+            ResourceDatabasePopulator populator = new ResourceDatabasePopulator();
```
Comment (alfonsomunozpomer) [inline src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java]: `var`.

File: src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java [hunk 4]
```diff
@@ -23,6 +23,48 @@
...
+            populator.addScripts(new ClassPathResource("fixtures/experiment-delete.sql"));
+            populator.execute(dataSource);
+        }
+
+        @ParameterizedTest
+        @MethodSource("expressionAtlasExperimentsProvider")
+        void testParserForExpressionAtlas(String experimentAccession) {
+            SdrfParser sdrfParser = new SdrfParser(new DataFileHub(dataFilesPath.resolve("gxa")));
```
Comment (alfonsomunozpomer) [inline src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java]: `var`.

File: src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java (hunk 1 above)
Comment (lingyun1010) [reply src/test/java/uk/ac/ebi/atlas/experimentimport/sdrf/SdrfParserIT.java]: Deleted.

**Pull request merged**